vcloud.yaml

```diff
@@ -1,11 +1,334 @@
+      `/transcribe/result` requests return a URL with the results as a zip or JSON file.
+  - name: Status Request
+    description:
+      The V-Cloud API is primarily used to transcribe audio and get the results,
+      but can also be used to determine if V-Cloud is available by querying the `/status`
+      endpoint.
 paths:
-  /:
+  /transcribe:
+    post:
+      tags:
+        - Transcribe Request
+      summary: Specify an audio file or URL source to transcribe
+      description:
+        Requests to the `/transcribe` endpoint must include the `file`
+        (or `url` for external audio sources) and `token` parameters. Additional [transcription
+        parameters](https://docs.vocitec.com/en/v-cloud-transcription-parameters.html)
+        are optional.
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              $ref: "#/components/schemas/transcription-parameters"
+        required: true
+      responses:
+        200:
+          $ref: "#/components/responses/requestid-success"
+        400:
+          $ref: "#/components/responses/code-400"
+        401:
+          $ref: "#/components/responses/code-401"
+        405:
+          $ref: "#/components/responses/code-405"
+        500:
+          $ref: "#/components/responses/code-500"
+        502:
+          $ref: "#/components/responses/code-502"
+        504:
+          $ref: "#/components/responses/code-504"
+  /transcribe/result:
+    get:
+      tags:
+        - Results Request
+      summary: Retrieve a secure link to transcription results
+      description:
+        Requests to the `/transcribe/result` endpoint must include a valid
+        `requestid` and `token` as query parameters. The format of your results depends
+        on the options specified in the initial `/transcribe` request.
+      parameters:
+        - $ref: "#/components/parameters/requestid"
+        - $ref: "#/components/parameters/token"
+      responses:
+        200:
+          $ref: "#/components/responses/results-success"
+        202:
+          $ref: "#/components/responses/code-202"
+        401:
+          $ref: "#/components/responses/code-401"
+        405:
+          $ref: "#/components/responses/code-405"
+        422:
+          $ref: "#/components/responses/code-422"
+        500:
+          $ref: "#/components/responses/code-500"
+        503:
+          $ref: "#/components/responses/code-503"
+        504:
+          $ref: "#/components/responses/code-504"
+  /status:
     get:
-      summary: Placeholder
+      tags:
+        - Status Request
+      summary: Check V-Cloud availability
+      description:
+        The response indicates the status and availability of V-Cloud.
+        Contact support@vocitec.com if you are experiencing an operational issue.
       responses:
         200:
-          description: Success
+          description: V-Cloud is available. You do not need to pass a token to make this call.
```

Comment:
Remove the `You do not need` portion.  That's applicable to the request, not the response.